icons/insomnia/insomnia-original-wordmark.svg

```diff
@@ -0,0 +1,14 @@
+ 
+</svg>
```

Comment:
Resized and centered using [boxy-svg](https://boxy-svg.com/app) and optimized using [SVGOMG](https://jakearchibald.github.io/svgomg/)
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128">
  <defs>
    <linearGradient id="a" x1="16.181" x2="16.181" y1="28.39" y2="5.61"
      gradientTransform="matrix(1.05738 0 0 1.05882 .43 45.999)" gradientUnits="userSpaceOnUse">
      <stop offset="0" stop-color="#7400e1" />
      <stop offset="1" stop-color="#4000bf" />
    </linearGradient>
  </defs>
  <path fill="#fff"
    d="M17.348 80.078c8.867 0 16.059-7.2 16.059-16.078 0-8.879-7.192-16.078-16.059-16.078S1.289 55.122 1.289 64c0 8.879 7.192 16.078 16.059 16.078Zm0 0" />
  <path fill="#4000bf"
    d="M17.348 47.059C8.004 47.059.43 54.645.43 64c0 9.355 7.574 16.941 16.918 16.941S34.266 73.355 34.266 64c0-9.355-7.574-16.941-16.918-16.941Zm0 1.722c8.395 0 15.2 6.813 15.2 15.219 0 8.406-6.806 15.219-15.2 15.219-8.395 0-15.2-6.813-15.2-15.219 0-8.406 6.805-15.219 15.2-15.219Zm0 0" />
  <path fill="url(#a)" fill-rule="evenodd"
    d="M17.54 51.941c6.652 0 12.042 5.399 12.042 12.059 0 6.66-5.39 12.059-12.043 12.059-6.652 0-12.043-5.399-12.043-12.059 0-1.629.32-3.18.907-4.598a4.672 4.672 0 0 0 3.777 1.918 4.69 4.69 0 0 0 2.77-8.472 11.946 11.946 0 0 1 4.59-.907Zm0 0" />
  <path fill="#fff"
    d="M50.563 56.492h-2.086v13.86h2.086v-13.86Zm4.902 7.688c0-1.657 1.012-2.606 2.418-2.606 1.371 0 2.203.903 2.203 2.41v6.368h2.024v-6.61c0-2.574-1.414-3.922-3.535-3.922-1.563 0-2.583.727-3.063 1.828h-.129v-1.691h-1.941v10.395h2.023V64.18Zm17.133-1.684c-.418-1.617-1.684-2.676-3.926-2.676-2.347 0-4.008 1.239-4.008 3.082 0 1.473.891 2.457 2.836 2.891l1.758.383c1 .222 1.469.672 1.469 1.32 0 .805-.86 1.438-2.184 1.438-1.21 0-1.988-.524-2.23-1.547l-1.953.3c.336 1.84 1.863 2.875 4.195 2.875 2.508 0 4.246-1.332 4.246-3.214 0-1.47-.933-2.375-2.84-2.817l-1.648-.379c-1.145-.27-1.637-.656-1.63-1.359-.007-.797.86-1.367 2.009-1.367 1.258 0 1.836.695 2.074 1.394l1.832-.324Zm6.797 8.067c2.926 0 4.84-2.145 4.84-5.36 0-3.234-1.914-5.383-4.84-5.383s-4.84 2.149-4.84 5.383c0 3.215 1.914 5.36 4.84 5.36Zm.008-1.7c-1.914 0-2.805-1.672-2.805-3.668 0-1.988.89-3.68 2.805-3.68 1.898 0 2.789 1.692 2.789 3.68 0 1.996-.89 3.668-2.79 3.668h.001Zm7.086 1.489h2.023v-6.407c0-1.402.98-2.379 2.129-2.379 1.121 0 1.898.746 1.898 1.875v6.91h2.016v-6.624c0-1.25.762-2.16 2.082-2.16 1.066 0 1.945.597 1.945 2.003v6.782h2.02v-6.97c0-2.374-1.324-3.562-3.203-3.562-1.492 0-2.617.72-3.117 1.828h-.106c-.453-1.136-1.406-1.828-2.793-1.828-1.371 0-2.39.684-2.824 1.828h-.129v-1.691h-1.941v10.395Zm18.843-6.172c0-1.657 1.016-2.606 2.418-2.606 1.375 0 2.203.903 2.203 2.41v6.368h2.024v-6.61c0-2.574-1.414-3.922-3.535-3.922-1.563 0-2.582.727-3.063 1.828h-.129v-1.691h-1.937v10.395h2.02l-.001-6.172Zm9.356 6.172h2.02V59.957h-2.02v10.395Zm1.02-12c.695 0 1.277-.54 1.277-1.204 0-.664-.582-1.21-1.278-1.21-.703 0-1.277.546-1.277 1.21 0 .665.574 1.204 1.277 1.204h.001Zm6.738 12.23c1.718 0 2.683-.871 3.07-1.648h.082v1.418h1.973v-6.903c0-3.023-2.38-3.629-4.028-3.629-1.879 0-3.61.758-4.285 2.657l1.899.43c.296-.735 1.054-1.446 2.414-1.446 1.304 0 1.972.684 1.972 1.86v.046c0 .738-.758.727-2.625.942-1.965.23-3.98.746-3.98 3.105 0 2.047 1.535 3.168 3.508 3.168Zm.441-1.625c-1.144 0-1.969-.512-1.969-1.516 0-1.082.961-1.468 2.13-1.625.655-.086 2.21-.261 2.503-.554v1.34c0 1.234-.98 2.355-2.664 2.355Zm0 0" />
</svg>
```